draft-ietf-httpbis-http2bis.xml

```diff
@@ -71,7 +71,7 @@
       <name>Introduction</name>
       <t>
         The Hypertext Transfer Protocol (HTTP) is a wildly successful protocol. However, the way
-        HTTP/1.1 uses the underlying transport (<xref target="RFC7230" section="6"/>) has several
+        HTTP/1.1 uses the underlying transport (<xref target="HTTP11"/>) has several
```

Comment:
We really don't have a section reference here?

Reply:
It's a very broad reference, it doesn't need to be precise.